src/types.ts

```diff
@@ -605,7 +605,7 @@ export type RevolutPayEventPayload<
 
 export interface PaymentsModuleRevolutPayInstance {
   mount: (
-    target: string | HTMLElement,
+    target: string | HTMLElement | null,
```

Comment:
don't we need the target to attach it to the DOM? Otherwise how do you know where to render the component?